File: appMain/src/main/java/com/cartlc/tracker/data/DataPictureCollection.java
package com.cartlc.tracker.data;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by dug on 5/15/17.
 */

public class DataPictureCollection {
    public long id; // collection id shared by all pictures.
    public List<DataPicture> pictures = new ArrayList();

    public DataPictureCollection(long id) {
        this.id = id;
    }

    public void add(DataPicture picture) {
        pictures.add(picture);
    }
}

File: appMain/src/main/java/com/cartlc/tracker/fresh/model/core/data/DataPictureCollection.kt
/**
 * Copyright 2018, FleetTLC. All rights reserved
 */
package com.cartlc.tracker.fresh.model.core.data

/**
 * Created by dug on 5/15/17.
 */

class DataPictureCollection(
        var id: Long // collection id shared by all pictures.
) {
    var pictures: MutableList<DataPicture> = mutableListOf()

    fun add(picture: DataPicture) {
        pictures.add(picture)
    }
}
